ClientLib/src/main/java/com/o3dr/android/client/Drone.java

```diff
@@ -481,7 +481,11 @@ public void generateDronie() {
     }
 
     public void arm(boolean arm) {
-        DroneStateApi.arm(this, arm);
+        DroneStateApi.arm(this, arm, false);
+    }
+
+    public void arm(boolean arm, boolean emergencyDisarm) {
```

Comment:
@squilter No need to add a corresponding method in the `Drone` class. Their uses are being deprecated (should be in latest from `develop`).